pkg/server/backuprestoreserver.go

```diff
@@ -224,22 +224,22 @@ func (b *BackupRestoreServer) runServer(ctx context.Context, restoreOpts *brtype
 			}
 			_ = miscellaneous.SleepWithContext(ctx, retryTimeout)
 		}
-	} else {
-		// when OriginalClusterSize = 1
-		m := member.NewMemberControl(b.config.EtcdConnectionConfig)
-		err := retry.OnError(retry.DefaultBackoff, func(err error) bool {
-			return err != nil
-		}, func() error {
-			cli, err := etcdutil.NewFactory(*b.config.EtcdConnectionConfig).NewCluster()
-			if err != nil {
-				return err
-			}
-			_, err = m.UpdateMemberPeerURL(ctx, cli)
-			return err
-		})
+	}
+
+	// when OriginalClusterSize = 1
+	m := member.NewMemberControl(b.config.EtcdConnectionConfig)
+	err := retry.OnError(retry.DefaultBackoff, func(err error) bool {
+		return err != nil
+	}, func() error {
```

Comment:
```suggestion
	err := retry.OnError(retry.DefaultBackoff, AnyError, func() error {
```
Please see [here](https://github.com/gardener/etcd-backup-restore/blob/master/pkg/errors/error.go#L38)

Reply:
Ah yes, i forgot. Changed.